tests/helpers.py

```diff
@@ -373,11 +376,19 @@ def delete_cephblockpool():
 
 def create_cephfilesystem():
     """
-    Function for deploying CephFileSystem (MDS)
+    Function for deploying CephFileSystem (MDS). If CephFilesystem exists
+    deployment will be skipped
 
     Returns:
         bool: True if CephFileSystem creates successful
     """
+    POD = pod.get_all_pods(
```

Comment:
What if CephFS exists from a previous test case as a leftover? I think we want to know about it and not to skip the creation. Maybe let the test the call this function decide wether the CephFS should exist or not.

Reply:
Hi @ebenahar Is it ok if we pass an arg to create_cephfilesystem like force_create so if it's true we can delete old mds and create a new one

Reply:
@prsurve sounds good

Reply:
will work on this today

Reply:
done